accel-pppd/radius/radius.c

```diff
@@ -1006,6 +1017,52 @@ static int load_config(void)
 	return 0;
 }
 
+static void ev_radius_coa_vrf(struct ev_radius_t *ev)
```

Comment:
since more than only VRF can be parsed, ev_radius_coa() name is ok.
using it in radius.c doesn't seem valid tho. dm_coa request/disconnect are all started in dm_coa(), even is used for foreign plugins, so I think it will be better to move it into dm_coa too